tests/test_cli_client.py

```
@@ -778,6 +897,59 @@ def handler(request):
+            return httpx.Response(
+                200,
+                json={
+                    "valid": True,
+                    "username": "User",
+                    "message": "Authenticated as User",
+                },
+            )
+
+        client = _make_client(handler)
+        result = client.validate_token(
+            token_type="jira",
```

Ruff (0.15.9)

[error] Possible hardcoded password assigned to argument: "token_type"

(S106)